packages/node/src/evm/transaction-counts.test.ts

```diff
@@ -14,16 +14,16 @@ import { ethers } from 'ethers';
 import * as transactions from './transaction-counts';
 import * as wallet from './wallet';
 
-describe('fetchByRequesterIndex', () => {
-  it('calls getTransactionCount once for each unique requester index', async () => {
+describe('fetchBySponsor', () => {
+  it('calls getTransactionCount once for each unique sponsor', async () => {
     getTransactionCountMock.mockResolvedValueOnce(5);
     const options = {
       currentBlock: 10716084,
       masterHDNode: wallet.getMasterHDNode(),
       provider: new ethers.providers.JsonRpcProvider(),
     };
     const indices = ['1', '1'];
-    const [logs, res] = await transactions.fetchByRequesterIndex(indices, options);
+    const [logs, res] = await transactions.fetchBySponsor(indices, options);
```

Comment:
The `indices` variable should be updated to be `sponsorAddresses` and ideally use actual addresses for clarity. Applies to all tests here

Reply:
Good catch 🎉